test/e2e/kubectl/portforward.go

```diff
@@ -49,7 +49,7 @@ const (
 
 // TODO support other ports besides 80
 var (
-	portForwardRegexp = regexp.MustCompile("Forwarding from (127.0.0.1|\\[::1\\]):([0-9]+) -> 80")
+	portForwardRegexp = regexp.MustCompile("Forwarding from 127.0.0.1:([0-9]+) -> 80")
```

Comment:
Why do we remove ipv6 localhost from matching?

Reply:
I was looking into flaky tests originally and thought that we are not going to need this after my fix, but then I realised that I was wrong (I thought it has something to do with the order of output).

I removed it.